candig/server/sqlite_backend.py

```diff
@@ -29,7 +26,7 @@ def sqliteRowToDict(sqliteRow):
     :param sqliteRow: single row returned from fetchone DB call
     :return: dictionary corresponding to this row
     """
-    return dict(zip(sqliteRow.keys(), sqliteRow))
+    return dict(list(zip(list(sqliteRow.keys()), sqliteRow)))
```

Comment:
dict-like-object to list, to zip, to list and to dict...ahhhh

Reply:
This is probably a bug with the auto-converter

Reply:
Replaced with `return dict(list(zip(sqliteRow.keys(), sqliteRow)))`